charts/operator-wandb/charts/app/values.yaml

```diff
@@ -74,6 +74,7 @@ artifactsGc:
   NumWorkers: 0
   DeleteFilesNumWorkers: 0
 
+# internalJWTMap is used to map service accounts to their issuers.
```

Comment:
Should I make a comment here to outline how people can fill this in depending on their cluster?

Reply:
Probably would help.  do we need to be setting this in our terraform?

Reply:
We're setting it internally here: https://github.com/wandb/core/pull/25402/files#diff-768fd3d816a568ef4f97bd468a85b8b652eb4aa9b3343a813e511d514b5aa916R307-R310

Reply:
Added a comment indicating the kubectl command you can use to get this for your cluster.